Cloudinary/Classes/Core/Features/Uploader/Results/CLDUploadResult.swift

```diff
@@ -165,6 +165,13 @@ import Foundation
         return CLDInfo(json: info)
     }
     
+    open var qualityAnalysis: CLDQualityAnalysis? {
```

Comment:
Please rename and bring back the older overload (the one returning a dictionary), so we don't break the code for people using it.

Reply:
name updated